api/proto/metrics/metrics.proto

```diff
@@ -76,6 +76,7 @@ message appCpuMetric {
   // deprecated = 3;
   google.protobuf.Timestamp upTime = 4;
   uint64 total = 5;                   // cpu total in secs.
+  uint64 systemTotal = 6;             // docker host system cpu total in secs. inc user, system and idle
```

Comment:
Does "total" include "systemTotal" or not? Would be good to clarify that on the line above.
And when there are multiple containers/modules being reported, do they all report the same systemTotal? The modules would be running inside the same VM image AFAIU

Reply:
No. I'm reuse this appCpuMetric proto API here. this is only container (user) usage portion of the total. I can make more comments on the line.
Yes, they will report the same systemTotal, as far as I can tell, so we can calculate the percentage usage of the container.